### DISCUSSION TITLE
kangaroo/integrate/integrate-ws/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2063] [Announcements]
# kangaroo/integrate/integrate-ws/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kangaroo/integrate/integrate-ws/

**Comment by liyuerich** [comment]
支持关联 Harbor 和 Docker 两种仓库实例

现在已经支持JFrog 集成，文档中能否补充？

**Comment by ModetaNiu** [comment]
Already fixed in PR #2106
